src/base/tristatebool.h

```diff
@@ -29,25 +29,32 @@
 #ifndef TRISTATEBOOL_H
 #define TRISTATEBOOL_H
 
+class QJsonValue;
+
 class TriStateBool
```

Comment:
wanna ask, is this class serves the same purpose as `boost::optional<bool>`?

ps. `boost::optional` is header only
pps. c++17 has `std::optional`